libcodechecker/server/client_db_access_server.py

```diff
@@ -120,10 +125,13 @@ def check_auth_in_request(self):
         return success
 
     def do_GET(self):
-        authToken = self.check_auth_in_request()
-        if authToken:
+        authSession = self.check_auth_in_request()
```

Comment:
Almost each time when we query a session we also extract the token. Maybe it would be cleaner to return both of these?